media/js/firefox/new/yandex/scene1.js

```diff
@@ -0,0 +1,231 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var Yandex = {
+        RUSSIA_COUNTRY_CODE: 'ru',
```

Comment:
The endpoint returns an uppercase `US` when I use it right now. I assume the `RU` will be uppercase too. Can you verify and change it? OR convert all the strings to lowercase before you do your comparison?

Reply:
This is converted to lowercase inside the `xhr.onload` callback, so it should be fine I think:

```javascript
country = JSON.parse(r.target.responseText).country_code.toLowerCase();
```

Reply:
💯